quay-enterprise/tectonic/files/quay-servicetoken-role-k8s1-6.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: Role
+metadata:  
+  name: quay-enterprise-serviceaccount
+  namespace: quay-enterprise
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - secrets
+  verbs:
+  - get
+  - put
+  - patch
+  - update
+- apiGroups:
+  - ""
+  resources:
+  - namespaces
```

Comment:
On a "Role" this isn't going to work since namespaces aren't a namespaced resource. What's the actual requirement here? Being able to see the namespace it's working in?

You do get the ability to get your own namespace just by creating this role though (that's a bit hidden in the RBAC docs). Is that enough?